Add argument to download script

oregonpillow opened the pull request:
Closes #1 

Just a suggestion, i found it useful to just enter the exact command than having to answer the prompt.  I think the prompt is nice as a fall back though for when no argument given.

oregonpillow committed:
update script

oregonpillow committed:
formatting

chuckn246 reviewed (changes requested):
I like your idea and will approve with a few changes.

- Revert changes to the colored output functions - keep them one line please.
- Same for the `1 | 2)` entry in the case statement
- Capitalize the s in the `# script arg` comment
- Wrap the `$arch_select` tests in curly braces (`${arch_select}`) 

Other than those personal preferences, I think it looks good!

Thanks for contributing and fixing my whitespace and backslash issues!

oregonpillow committed:
validate seed file

oregonpillow committed:
revert change

oregonpillow committed:
format changes

oregonpillow committed:
revert change

oregonpillow commented:
Ok i _think_  i've addressed your points. Let me know if something still isn't fixed.
I was about to additionally make changes to the mkiso but realised i should save those for another issue/MR...

chuckn246 commented:
> Ok i _think_ i've addressed your points. Let me know if something still isn't fixed. 

Checking now!

> I was about to additionally make changes to the mkiso but realised i should save those for another issue/MR...

Yes please create another request for those changes, I was just about to mention that :)

chuckn246 commented on get_latest_iso:
Looks good, thank you for fixing my requests
